stream-chat-android-ui-components/src/main/res/drawable/stream_ic_command.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+    <path
+        android:fillColor="#006CFF"
```

Comment:
Same question as here https://github.com/GetStream/stream-chat-android/pull/817#discussion_r519785377, should we stick with black by default, and then tint as needed?

Reply:
Not that simple. It is a bit verbose to set tint for `TextView` with drawable as you need to do it programmatically. And in the icon below with 2 colors it seems impossible to apply tint. Lets leave it as is.

Reply:
True, definitely can't do this for multi-colour icons. 

Doesn't [this `drawableTint` solution](https://stackoverflow.com/a/57899724/4465208) do what we need for single-colour icons though?

Reply:
Yep, something was wrong with my android studio. It works. Also added tint to `stream_ic_mention.xml`. Modified multi-color icon `stream_ic_command_circle.xml` to reference internally the blue color I extracted as color res.